tests/conftest.py

```diff
@@ -248,33 +248,35 @@ def patched_get_exit_signal(pair, timeframe, dataframe, is_short):
     freqtrade.exchange.refresh_latest_ohlcv = lambda p: None
 
 
-def create_mock_trades(fee, is_short: bool = False, use_db: bool = True):
+def create_mock_trades(fee, is_short: Optional[bool] = False, use_db: bool = True):
```

Comment:
I think this should either be true or false, not Optional, unless there's a good reason for it being optional

Reply:
Ya, we should always specify `is_short=False/True`  when passing in the parameters...  just in case it went to use_db.

Reply:
Well as it is, I think that it will always default to `False` (not None) which is kind of weird behavior

Reply:
> Ya, we should always specify `is_short=False/True` when passing in the parameters... just in case it went to use_db.

Either we should do that, or `is_short` should be the last parameter

Reply:
This is optional because `None` has a meaning, which will result in a mix of buys and sells.